### PR TITLE
Bump plugint o Playframework 2.7

### PR DESCRIPTION
**Pull request by LukaszByczynski**
It was easy job to do. I only performed plugins upgrade, some code cleanup and added code formatter.

### CODE REVIEW

**Comment by LukaszByczynski** [comment]
@tanin47  I'm waiting for a review from you.

File: test-play-project/project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.0-RC3
```
Comment (tanin47) [inline test-play-project/project/build.properties]: Is it possible to avoid using the RC version?

Comment (LukaszByczynski) [reply test-play-project/project/build.properties]: No problem I can revert back to 1.2.8. (I intentionally added this version to be sure that plugin works with 1.3.0 - this version it's just at corner)

**Comment by tanin47** [comment]
4.1.0 has been published. Thank you again for bumping the version and adding scalafmt.

Please test and let me know if it works well for you.

**Comment by LukaszByczynski** [comment]
Yes, it's works correctly. In the next few days, I'm going to push small fixes to the test project build process (removes unnecessary .vue files from production build) 

I still wonder about #20. I hadn't found any issues after changing css styles. Components were recompiled correctly. What you think about it?

**Pull request merged**